core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -51,7 +51,8 @@ object ConsoleConsumer extends Logging {
 
     val consumer =
       if (conf.useNewConsumer) {
-        new NewShinyConsumer(conf.topicArg, getNewConsumerProps(conf))
+        val timeoutMs = if (conf.timeoutMs >= 0) conf.timeoutMs else Long.MaxValue
```

Comment:
@auradkar The timeout parameter is consistent with "consumer.timeout.ms" config option in the old consumer. With the old consumer, timeout less than zero indicates that there is no timeout. The same behaviour is used here for consistency.